sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -1066,3 +1066,123 @@ test_suite(
     ],
     tests = ["asm_interrupt_handler_fpga_cw310_test_otp_" + otp_name for otp_name in OTP_CFGS_EXEC_DISABLED],
 )
+
+# Shutdown Redact Test
+
+# Generate special OTP images
+# For DEV, PROD, and PROD_END, cycle through ROM_ERROR_REPORTING OTP item
+
+LC_STATES = [
+    "test_unlocked0",
+    "rma",
+    "dev",
+    "prod",
+    "prod_end",
+]
+
+# The LC states in which redaction is enabled. In all other states, there
+# should be no redaction.
+REDACTION_LC_STATES = [
+    "dev",
+    "prod",
+    "prod_end",
+]
+
+ERROR_REDACT_VALUES = {
+    "none": "0xe2290aa5",
+    "error": "0x3367d3d4",
+    "module": "0x1e791123",
+    "all": "0x48eb4bd9",
+}
+
+BFV_VALS = {
+    "none": "0142500d",
+    "error": "0042500d",
+    "module": "0000000d",
+    "all": "ffffffff",
+}
```

Comment:
nit: I think we can slightly clean up this part, e.g. move the constants to a `.bzl` file, and maybe implement the redaction as a bazel macro so that we can change test expectations based on the OTP config but I'd prefer merging this since other PRs will use the OTP updates in this PR.

Reply:
Done in #15928